README.md

```diff
@@ -54,6 +54,77 @@ to you z config:
 messages:
     overwrite_compatibility: false
 ```
+
+## Automatic translation
+Leverage the `zicht:message:translate` command with automatic translation through an API of choice.
+Kickstart the usage by using the provided Google Translate API.
+
+### Setup
+
+`composer require google/cloud-translate`
+
+### Configuration
+Define the Google Translator as a service in your project. Your API key should be the private key from a service account (https://cloud.google.com/translate/docs/basic/setup-basic)
+
+```yaml
+parameters:
+    env(GOOGLE_API_KEY): '%kernel.root_dir%/config/your-google-api-key.json'
+```
+```xml
+<service class="Zicht\Bundle\MessagesBundle\Translator\GoogleTranslator">
```

Comment:
Ik heb het fout geformuleerd in eerdere comments. Als de ID hetzelfde is als de class, dan mag je de class weglaten:

```xml
<service id="Zicht\Bundle\MessagesBundle\Translator\GoogleTranslator">
```

Reply:
:trollface: